### ISSUE TITLE
github warning: app uses deprecated authorizations endpoint on github api

### ISSUE THREAD

**Issue by robbyjm** [mattn/vim-gist#227]
On April 14th, 2020 at 22:12 (UTC) you or an application you used recently accessed the deprecated Authorizations endpoint on the GitHub API with the useragent curl/7.66.0.

We will remove the Authorizations API endpoint on November 13, 2020. If you accessed the API via password authentication, then we recommend you use the web flow to authenticate. Please check that your app uses the web flow for authentication https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps/#web-application-flow

You can learn more about these changes by visiting our developer blog https://developer.github.com/changes/2020-02-14-deprecating-oauth-auth-endpoint

Thanks,
The GitHub Team

**Comment by ypcrts** [comment]
Yeah, same. It would be nice to have something in the docs as a manual workaround, so we can generate `~/.gist-vim` and put `token 923749028374098273490827` in there.

**Comment by mattn** [comment]
Thanks. Will look later

**Comment by hholst80** [comment]
Whatever it was doing it is no longer working.

> This API can only be accessed with username and password Basic Auth

**Comment by mattn** [comment]
Yes, only possible to do is removing password prompt.
